sipa/utils/__init__.py

```diff
@@ -82,6 +88,38 @@ def support_hotline_available():
         return False
 
 
+@cached(cache=TTLCache(maxsize=1, ttl=300))
+def meetingcal():
+    url = "https://agdsn.de/cloud/remote.php/dav/public-calendars/bgiQmBstmfzRdMeH?export"
```

Comment:
This URL should ideally be configurable.
But given that the support hotline is also hardcoded (not intentionally, but the value is statically imported from the default config and thus hard-coded) we can ignore that for now and maybe address it in a separate issue.

Reply:
See #443.

@jnsga please resolve this conversation just to communicate that you've seen this, there's nothing to do for you here.